services/madoc-ts/src/utility/linking-property-types.ts

```diff
@@ -30,11 +30,3 @@ export function isLinkCaptureModelParagraphs(link: ResourceLinkResponse) {
     link.link.profile === PARAGRAPHS_PROFILE
   );
 }
-
-export function isLinkLocalOcr(link: ResourceLinkResponse) {
-  return isLinkPlaintext(link) || isLinkCaptureModelParagraphs(link);
-}
-
-export function isLinkOcr(link: ResourceLinkResponse) {
-  return isLinkPlaintext(link) || isLinkHocr(link) || isLinkMetsAlto(link);
-}
```

Comment:
I'm surprised we don't use these! Fine to delete though

Reply:
So I used ts-prune to find unused code, but there were quite a lot of false positives in the list it gave me. I ended up going through the list and checking in Intellij (which is better at detecting if code is actually used or not, but doesn't have a view to show all instances of unused code), and these were still unused. Happy to put these back in if you think they might be useful in the future though

Reply:
No, should be fine to remove. I think we use the more granular ones that this uses